docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: "Learning Dagster from Airlfow"
+description: How to get started with Dagster from an Airflow background
+---
+
+# Learning Dagster from Airflow
+
+In this tutorial, we'll help you make the switch from Airflow to Dagster. Here, we review an Airflow DAG and show how the same functionality and be achieved in Dagster.
```

Comment:
This should be `and show how the same functionality can be achieved in Dagster.`